Makefile

```diff
@@ -193,9 +193,10 @@ grapl-template-generator: ## Build the Grapl Template Generator and install it t
 		./bin/grapl-template-generator
 	printf -- '\n${FMT_BOLD}Template Generator${FMT_END} written to ${FMT_BLUE}./bin/grapl-template-generator${FMT_END}\n'
 
-.PHONY: dump-artifacts
-dump-artifacts:  # Run the script that dumps Nomad/Docker logs after test runs
-	./pants run ./etc/ci_scripts/dump_artifacts --run-args="--compose-project=${COMPOSE_PROJECT_NAME}"
+.PHONY: dump-artifacts-local
+dump-artifacts-local:  # Run the script that dumps Nomad/Docker logs after test runs
+	./pants run ./etc/ci_scripts/dump_artifacts \
+		--run-args="--compose-project=${COMPOSE_PROJECT_NAME} --dump-agent-logs=True"
```

Comment:
FYI, you can use the old `--` trick instead of shoehorning everything into a `--run-args` value if you like:

```
./pants run ./etc/ci_scripts/dump_artifacts -- --compose-project=... --dump-agent-logs=True
```